src/main/java/io/antmedia/datastore/db/DataStore.java

```diff
@@ -531,6 +531,15 @@ public List<SubscriberStats> listAllSubscriberStats(String streamId, int offset,
 	 */		
 	public abstract boolean deleteSubscriber(String streamId, String subscriberId);
 
+	/**
+	 * deletes subscriber from the datastore for this stream
+	 * @param streamId
+	 * @param subscriberId - id of the subsciber to be deleted
+	 * @return- true if set, false if not
+	 */
+	public abstract boolean blockSubscriber(String streamId, String subscriberId, boolean playBlocked, boolean publishBlocked, long playBlockTime,
```

Comment:
Please just define the time where blocking happens in unixtimestamp format. So that we don't need `playBlocktime`, `publishBlockTime`

Reply:
Users have the option to prevent individuals from engaging in both playback and publishing activities, each with its distinct time interval. This requirement is accommodated through two separate variables: playBlockTime and publishBlockTime.

Reply:
This implementation seems hard to implement and I don't think it's required. Just block the player and publisher to some specific time in unixtimestamp. I mean publishedBlockedUntilTime and playBlockedUntilTime are enough